docs/using/recipe_code/custom_environment.dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM jupyter/base-notebook
 
 # name your environment and choose the python version
-ARG env_name=python38
-ARG py_ver=3.8
+ARG env_name=python310
+ARG py_ver=3.10
```

Comment:
3.8 also works fine.
I updated these lines to make sure 3.10 also works.
And this recipe will stay up-to-date a bit longer (till 3.10 reaches eol).